app/javascript/pages/Admin/Search/Purchases/Index.tsx

```diff
@@ -0,0 +1,167 @@
+                    <ul className="inline">
+                      <li>{purchase.purchase_state}</li>
+                      {purchase.stripe_refunded ? <li>(refunded)</li> : null}
+                      {purchase.stripe_partially_refunded ? <li>(partially refunded)</li> : null}
+                      {purchase.chargedback ? (
+                        <li>(chargeback{purchase.chargeback_reversed ? " reversed" : ""})</li>
+                      ) : null}
+                      {purchase.error_code ? (
+                        <li>
+                          {purchase.last_chargebacked_purchase ? (
+                            <Link href={Routes.admin_purchase_path(purchase.last_chargebacked_purchase)}>
+                              {purchase.error_code}
+                            </Link>
+                          ) : (
+                            purchase.error_code
+                          )}
+                        </li>
+                      ) : null}
+                    </ul>
+                    <div className="text-sm">
+                      <ul className="inline">
+                        {purchase.purchase_refund_policy ? (
+                          <li>
+                            Refund policy: {purchase.purchase_refund_policy}{" "}
+                            {purchase.product_refund_policy ? (
+                              <WithTooltip tip={`Current refund policy: ${purchase.product_refund_policy}`}>
+                                <Icon name="solid-shield-exclamation" className="text-warning" />
+                              </WithTooltip>
+                            ) : null}
+                          </li>
+                        ) : null}
+                        <li>
+                          Seller: {purchase.seller.email}{" "}
+                          <CopyToClipboard text={purchase.seller.email}>
+                            <Icon name="outline-duplicate" />
+                          </CopyToClipboard>
+                        </li>
+                        {purchase.seller.support_email ? (
+                          <li>Seller support email: {purchase.seller.support_email}</li>
+                        ) : null}
+                      </ul>
+                    </div>
+                  </td>
+                  <td data-label="By">
+                    <Link href={Routes.admin_search_purchases_path({ query: purchase.email })}>{purchase.email}</Link>{" "}
+                    <CopyToClipboard text={purchase.email}>
+                      <Icon name="outline-duplicate" />
+                    </CopyToClipboard>
+                    <small>
+                      <DateTimeWithRelativeTooltip date={purchase.created_at} />
+                    </small>
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+          <PaginatedLoader itemsLength={purchases.length} pagination={pagination} only={["purchases", "pagination"]} />
```

Comment:
Used infinite scrolling rather than explicit pagination here for consistency, but lmk if you disagree

Reply:
Search users is infinite scroll, so I’m fine with doing that here as well - either way we should keep them consistent IMO